lib/src/signed_video_h26x_sign.c

```diff
@@ -116,7 +110,6 @@ complete_sei_nalu_and_add_to_prepend(signed_video_t *self)
   // Get the oldest sei data
   const int sei_data_buffer_end = self->sei_data_buffer_idx;
   assert(sei_data_buffer_end <= MAX_NALUS_TO_PREPEND);
-  SignedVideoPrependInstruction prepend_instruction = SIGNED_VIDEO_PREPEND_NOTHING;
   size_t data_size = 0;
   svi_rc status = SVI_UNKNOWN;
   sei_data_t *nalu_to_prepend = &(self->sei_data_buffer[self->num_of_completed_seis]);
```

Comment:
I think you should rename `nalu_to_prepend` to for example `sei_data` now when we deal with sei_data_buffer instead.